server/src/main/java/com/kritsit/casetracker/server/datalayer/DatabasePersistence.java

```diff
@@ -53,11 +53,14 @@ public boolean isOpen() {
 
     private ResultSet get(String sql) throws SQLException {
         logger.info("Executing request {}", sql);
-        ResultSet query = null;
-        Statement statement = connection.createStatement(ResultSet.CONCUR_READ_ONLY, 
-                                                         ResultSet.TYPE_FORWARD_ONLY);
-        query = statement.executeQuery(sql);
-        return query;
+        
+        try(Statement statement = connection.createStatement(ResultSet.CONCUR_READ_ONLY, 
+                ResultSet.TYPE_FORWARD_ONLY)) {
+             ResultSet query = statement.executeQuery(sql);
+             return query;
+        } catch (SQLException e){
+            throw e;
+        }
     }
```

Comment:
Yes, it is complaining that there could be SQL injection as the input is from the user. So potentially prepared statements will prevent this, as well as being more optimised (see [Using Prepared Statements](http://docs.oracle.com/javase/tutorial/jdbc/basics/prepared.html))